### DISCUSSION TITLE
Pessoal vocês já pensaram em utilizar somente a área de discussões para postar vagas de ti/dúvidas/eventos/me-contrata/networking etc em único local?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2312] [Perguntas e Respostas] [answered]
Olá a todos!

Pessoal vocês já pensaram em utilizar somente a área de discussões para postar vagas de ti/dúvidas/eventos/me-contrata/networking etc em único local?

#### 1. motivos para isso
1. A gente tem somente 1 local para eventos, vagas, cursos e dúvidas. Isso é vantajoso, pois o usuário iniciante concentra e pesquisa somente em um local tais informações.
2. Isso poderia fortalecer a comunidade frontendbr, haja vista que em um único local as pessoas poderiam conhecer empresas, dúvidas comuns etc.
3. Atualmente a API de discussões pode ser integrada a telegram/slack etc. O que facilita a comunicação das pessoas em relação ao grupo.
4. Eu tenho uma tendência natural a sempre centralizar as coisas, acho mais fácil de encontrar coisas em um local específico. E isso pode ser uma vantagem ou erro.

#### 2. prova de conceito
![image](https://user-images.githubusercontent.com/95204402/175206711-8e3a718f-4977-4144-a6d1-ae8908f848ac.png)

#### 2.1 descrição da imagem
Como podemos ver na imagem podemos acompanhar as discussões de ti: vagas, eventos, perguntas/respostas, cursos.

**Comment by felquis** [comment, marked as answer]
eai parceiro, tudo bem?

Então essa organização foi dividida em alguns repositórios https://github.com/frontendbr/forum#reposit%C3%B3rios-da-front-end-brasil

Essa é a lista completa:

[Fórum](https://github.com/frontendbr/forum)
[Eventos](https://github.com/frontendbr/eventos)
[Vagas](https://github.com/frontendbr/vagas)
[Me Contrata](https://github.com/frontendbr/me-contrata)
[Compra e Venda](https://github.com/frontendbr/compra-e-venda)
[Doe um Livro](https://github.com/frontendbr/doe-um-livro)
[Poste Mais!](https://github.com/frontendbr/poste-mais)
[Open Source](https://github.com/frontendbr/open-source)
[Sugestões](https://github.com/frontendbr/sugestoes)

- - -

De fato sua sugestão é legal, e manteria todo mundo em um lugar só. Não sei se a divisão por repositórios é uma decisão antiga e que hoje as pessoas não tem muita vontade de mudar.

Mas é fato que a divisão ajuda a gente a ter um FOCO muito maior, do que ficar constantemente sendo notificado de novas discussões que não são discussões mas sim job-posting por exemplo.

O /vagas é particularmente bem movimento até hoje, por isso acredito que a divisão faz sentido e tem tido sucesso.

O que eu faço é me inscrever em todas as notificações do repositórios que eu tenho interesse em participar mais, assim fica muito mais fácil receber as notificações e entrar se aparecer algo que é do meu interesse.

Se alguém postar uma vaga ou evento aqui neste forum, eu vou provavelmente apontar para o repositório correto e bloquear novos comentários na discussão, ai a pessoa pode abrir outra discussão para reclamar ou para dar a opinião.

Isso ai é minha visão, é nós! o/
